custom/enikshay/nikshay_datamigration/tests/test_create_enikshay_cases.py

```diff
@@ -263,6 +263,18 @@ def test_case_update(self):
         episode_case = self.case_accessor.get_case(episode_case_ids[0])
         self.assertEqual(episode_case.dynamic_case_properties()['disease_classification'], 'extra_pulmonary')
 
+    @run_with_all_backends
+    def test_location_not_found(self):
+        self.loc.delete()
+        call_command('create_enikshay_cases', self.domain.name)
+
+        person_case_ids = self.case_accessor.get_case_ids_in_domain(type='person')
+        self.assertEqual(1, len(person_case_ids))
+        person_case = self.case_accessor.get_case(person_case_ids[0])
+        self.assertEqual(person_case.owner_id, '')
```

Comment:
Hmm. I think it might be better to set it to `'_archive_'` and set `person.archive_reason = 'migration_location_not_found'`